next.config.js

```diff
@@ -17,7 +17,8 @@ module.exports = {
         ]
       }
     ]
-  }
+  },
+  transpilePackages: ['dayjs']
```

Comment:
Why this?

Reply:
To make day.js typescript code be able to run on the browser https://github.com/BeautyyuYanli/blog/blob/39d02294b6a72aa401f3c27a16ac30ea34579423/components/FormattedDate.js#L6

Reply:
Ah I see.